float64 x_position
float64 y_position
float64 z_position
float64 yaw_angle
string tf_link
---
bool is_published